src/CoreVideo/CVTime.cs

```diff
@@ -43,15 +45,15 @@ public struct CVTime {
 		public int Flags { get { return (int) TimeFlags; } set { TimeFlags = (CVTimeFlags) value; }}
 
 #if !COREBUILD
-		public static CVTime ZeroTime {
+		public static CVTime? ZeroTime {
```

Comment:
This is a breaking change, because `CVTime` is a struct, so it becomes `Nullable<CVTime>` if you make it nullable.

Reply:
This was not needed after the addition of '!' good catch.